vite.config.js

```diff
@@ -0,0 +1,29 @@
+import { defineConfig } from 'vite';
+import { resolve } from 'path';
+import rollupOptions from './rollup.config';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+    base: '/pixi-shadows/',
```

Comment:
```suggestion
    base: '/pixi-shadows/dist/',
```

I have never used vite, so I'm not a 100% sure this is correct. However I tried testing locally using a static file server after having build the demo (and having removed the `/pixi-shadows` prefix here), and it seems `/dist` was required. It would make some sense for that to be required, since of course github-pages (which I use for hosting) will also serve just from the root of the project. 

We could also change the build dir to `docs` and then we can instruct github pages to host from that directory, then no modification to `base` is needed. Unfortunately we can't instruct github pages to hose from `dist` though.

Reply:
done.
I personally prefer not to use github paged custom behavior so that there is less magic and a local http server will work the same way. but not a strong opinion here.